src/components/dashboard/components/dictionary/styles/dictionary-modal.css

```diff
@@ -17,6 +17,8 @@
     padding-right: 8%;
     margin-left: 4%;
 }
+
```

Comment:
What happened here?

Reply:
I have removed that extra white space and updated the PR.